understanding/20/use-of-color.html

```diff
@@ -157,18 +170,27 @@ <h2>Examples of Use of Color</h2>
       <dl>
          <dt>A form that uses color and text to indicate required fields</dt>
          <dd>A form contains both required and optional fields. Instructions at the top of the
-               form explain that required fields are labeled with red text and also with an icon.
-               Users who cannot perceive the difference between the optional field labels and the
-               red labels for the required fields will still be able to see the icon next to the
-               red labels.</dd>
+             form explain that required fields are labeled with red text and also with an icon.
+             Users who cannot perceive the difference between the optional field labels and the
+             red labels for the required fields will still be able to see the icon next to the
+             red labels.</dd>             
          <dt>An examination</dt>
          <dd>Students view an SVG image of a chemical compound and identify the chemical elements
-               present based <strong>both</strong> on the colors used, as well as numbers next to each
-               element. A legend shows the color and number for each type of element. Sighted users who
-               cannot perceive all the color differences can still understand the image by relying on
-               the numbers.</dd>
+             present based <strong>both</strong> on the colors used, as well as numbers next to each
+             element. A legend shows the color and number for each type of element. Sighted users who
+             cannot perceive all the color differences can still understand the image by relying on
+             the numbers.</dd>
+         <dt>A series of toggle buttons</dt>
+         <dd>Users are presented with a series of toggle buttons that can be turned on or off. Visually,
+             when a toggle button is turned on, it not only has a different background color, but is
+             also styled to appear "pressed", with a distinctive inner shadow. Alternatively, toggles that
+             are turned on could have an extra "ticked" icon to distinguish them from toggles that are
+             turned off.</dd>
+         <dt>Focus indication</dt>
```

Comment:
but unarguably you're just relying on color here to convey something. so while non-text contrast can't compare two different states that are not adjacent, use of color can (because if the change is less than 3:1 it counts as only a change in color, which is a use of color)

Reply:
> but unarguably you're just relying on color here to convey something

Not unarguable, since that's exactly what I'm challenging! I think it is clear that a transition from nothing (blank white page) to _any_ content was not intended to be caught by Use of Color. By that definition, EVERYTHING added to or removed from a page is a failure of use of color, since the text, the symbols, the shadows, etc., is simply a use of color. Color alone is forming the letters. Color alone is forming the shapes. Color alone is forming the fills.
Clearly, Contrast (Minimum) was intended to specifically assess the contrast between text and its background. And Non-text Contrast was intended to assess contrast between any non-text visually present feature and its adjacent color (typically the page background).
I'm not trying to split hairs here. Your final note identifies the exact same problem.
> However, particularly in the case of state/focus indication, note that if the visual indicator was not previously present, this does not count as a change in color. For instance, if a focused element has a new border, or a new solid background shape, that was not there in its unfocused state, this does not count as a change of color alone, since the indicator was not present to begin with.

Reply:
getting lost between resolved comments, your myriad of tiny changes (could really do with doing them with proper git and pushing up a single commit, rather than piecemeal) etc. thought this comment was still about toggles, but i see it's the next one. and in this case, this was an example where it's not JUST a use of color change, and it's the additional outline that absolves the example from being a FAIL.

Reply:
if anything, it's the whole angle of these (existing) examples (which i then added to) ... even the existing ones aren't examples of failures, but examples of "this would fail, were it not for..."